docs_website/docs/changelog/2022-12-20.md

```diff
@@ -0,0 +1,165 @@
+---
+id: dec_2022_3_15_0
+title: Dec 2022 (version 3.15.0)
+sidebar_label: Dec 2022 (3.15.0)
+---
+
+Welcome to the latest release of Querybook 🎉.
+
+Following are the top new features we have added during the year of 2022:
+
+- **Query Search**: Users can search query exeuctions now.
+- **UDF Creator**: Custom UDF plugin can be added to let users add UDFs easily in the query editor.
+- **Schedules Page**: Users can see all your sechduled data docs in a central place now.
+- **Dynamic Result Size**: Users can now configure how many rows of results to display on the UI.
+- **Metastore Refresh**: Users can directly refresh a table from metastore manually.
+- **Column Filtering**: Toggle which columns to show.
+- **Table Uploader**: Create and upload a table from query exeuction result or a file.
+- **Customized Tags**: Annotate tags with features like color, tooltip, icon, rank and etc.
+- **List V2**: Updated the List feature.
+- **Automatic Query Limit**: Limit will be added to a select query without limits.
+- **Live Syntax Check**: Check the query syntax and show the errors in realtime
+- **Query Tanspiler**: Transpile queries between Presto and SparkSQL.
+- **Dag Exporter**: Export a DataDoc as a DAG to systems like airflow.
+- **Customized Schedule Notification**: Users can send customized notifications for scheduled docs.
+- **Event Logging**: Provides the instrumentation support of logging client action events, as well as API events.
```

Comment:
- **Event Logging**: Provides the instrumentation support of logging client action events and  API events.